pydiscourse/client.py

```diff
@@ -1453,7 +1547,7 @@ def _request(
         except ValueError:
             raise DiscourseError("failed to decode response", response=response)
 
-        if "errors" in decoded:
+        if "errors" in decoded and len(decoded["errors"]) > 0:
```

Comment:
I was about to merge when I noticed this one - quick question on why the additional check for length. Did you add it out of caution or have you encountered some condition where errors was populated but zero length?

Reply:
I don't really know. I took back this fork from my predecessor. I asked him for what route he needed this check but he don't remember.
I can delete it if you want :) Let me know !
Thanks !

Reply:
Please drop that change from this pr, we can review it separately after :-)

Reply:
Done ! I'll make another PR with more details when / if I find the root cause.
Thanks !